_sass/_hca.scss

```diff
@@ -38,24 +38,6 @@ body .row {
   }
 }
 
-.review-view {
```

Comment:
Yes it can be removed, it was from a previous implementation that we are no longer using